pack/wsp.json

```diff
@@ -93,6 +92,7 @@
 		"deck_limit": 2,
 		"faction_code": "hero",
 		"flavor": "\"Glad to be of service!\" - Wasp",
+		"health_per_hero": false,
```

Comment:
What does this property do on an **event**?

Reply:
Nothing, removed now. Thanks!